robot/web/stream/server.go

```diff
@@ -48,6 +49,7 @@ func (ss *streamState) Stop() {
 // Server implements the gRPC audio/video streaming service.
 type Server struct {
 	streampb.UnimplementedStreamServiceServer
+	robot robot.Robot
```

Comment:
why are we adding this? Where is it used?

Reply:
I'll take it out. I was testing this by slicing a `cam := camera.FromRobot(robot, ...)` inside the stream server's `AddStream`.